src/main/java/backend/Season.java

```diff
@@ -0,0 +1,87 @@
+package backend;
+
+import com.google.gson.Gson;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Season {
+    private int currentRound;
+
+    /**
+     * Create a new season.
+     *
+     */
+    public Season() {
+        this.currentRound = 0;
+        // Yin, je kan hier verder alles toevoegen,
+        // ik heb deze class alleen aangemaakt voor de json parser.
```

Comment:
Could you translate the Dutch comments to English. Makes more sense if the programming language is in English